api/app/tests/apps/users/v1/test_users.py

```diff
@@ -329,3 +329,40 @@ def test_history_view(self):
         self.assertEqual(change_log_data['who'], self.superuser.username)
         self.assertIn('Voornaam gewijzigd:\n Patched', change_log_data['action'])
         self.assertIn(f'Rol wijziging:\n {group.name}', change_log_data['action'])
+
+    def test_get_users_no_view_user_permission(self):
+        """
+        Check if a user that has no "view_user" rights is not able to retrieve the list of users
+        """
+        user = UserFactory.create()
```

Comment:
Nitpick, and not blocker: I would like it better if this was the same user as the rest of the tests in this test case, but without the `view_user` permission. That would more clearly demonstrate the need for only the `view_user` permission.

As-is the `user` and `self.sia_read_write_user` differ in two permissions (also the `sia_write` and `view_user`).

Reply:
Update: we discussed this. We will tackle users in test cases as a separate clean-up PR.